schema/verify/functions/age_conversion.sql

```diff
@@ -0,0 +1,30 @@
+-- Verify seattleflu/schema:functions/age_conversion on pg
+
+begin;
+
+do $$ 
+
+	declare
+    	age_in_months int;
+    	months int;
+    	years numeric;
+    	month_to_years numeric;
+
+	begin
+
+		with ages (age_as_interval, age_in_months) as (
+		    select make_interval(months => x), x from generate_series(0, 12) as x
+		    union
+		    select make_interval(years => x), x * 12 from generate_series(1, 6) as x
+		)		
+		select ages.age_in_months, age_in_months(age_as_interval), age_in_years(age_as_interval), age_month_to_years(ages.age_in_months) 
+		  into age_in_months, months, years, month_to_years
+		  from ages;
+  
+  		assert age_in_months = months;
+		assert years = round(age_in_months / 12::numeric, 2);
+		assert month_to_years = years;
```

Comment:
The test of `years` and `month_to_years` is effectively inlining the implementation of the function, which is not a useful test because they're not independent. It would be better to hardcode the test input values and expected results in a temporary table of tests. An example of this can be found in `schema/verify/functions/hamming_distance.sql`.